### PR TITLE
Update to newer version of Serilog that supports flushing

### PR DESCRIPTION
**Pull request by pakrym**
And use it.

Closes: https://github.com/aspnet/Logging/issues/497

### CODE REVIEW

File: src/Microsoft.Extensions.Logging.AzureAppServices/project.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "NETStandard.Library": "1.6.1-*",
     "Serilog.Extensions.Logging": "1.0.0",
     "Serilog.Sinks.PeriodicBatching": "2.0.0",
-    "Serilog.Sinks.RollingFile": "2.1.0",
+    "Serilog.Sinks.RollingFile": "3.1.0-dev-00745",
```
Comment (pranavkm) [inline src/Microsoft.Extensions.Logging.AzureAppServices/project.json]: Can you remove the `imports` with this change?

File: src/Microsoft.Extensions.Logging.AzureAppServices/AzureAppServicesDiagnosticsSettings.cs [hunk 2]
```diff
@@ -41,7 +41,14 @@ public class AzureAppServicesDiagnosticsSettings
         /// </summary>
         public string BlobName { get; set; } = "applicationLog.txt";
 
+        /// <summary>
         /// Gets of sets the maximum size of the background log message queue.
```
Comment (ryanbrandenburg) [inline src/Microsoft.Extensions.Logging.AzureAppServices/AzureAppServicesDiagnosticsSettings.cs]: `Gets or sets`

**Comment by nblumhardt** [comment]
Heads-up, [Serilog.Sinks.RollingFile](https://www.nuget.org/packages/serilog.sinks.rollingfile) 3.1.0 is now final.

**Comment by pakrym** [comment]
Updated.

**Pull request merged**